README.rst

```diff
@@ -133,6 +133,62 @@ input queries before sending them to the API:
 Run ``python -m autoextract --help`` to get description of all supported
 options.
 
+Errors
+~~~~~~
+
+The following errors could happen while making requests:
+
+- Network errors
+- `Request-level errors`_
+    - Authentication failure
+    - Malformed request
+    - Too many queries in request
+    - Request payload size is too large
+- `Query-level errors`_
+    - Downloader errors
+    - Proxy errors
+    - ...
+
+Some errors could be retried while others don't.
+
+For example,
+you could retry a query with a Proxy Timeout error
+because this is a temporary error
+and there are chances that this response will be different
+within the next retries.
+
+On the other hand,
+it makes no sense to retry queries that return a 404 Not Found error
+because the response is not supposed to change if retried.
+
+.. _Request-level errors: https://doc.scrapinghub.com/autoextract.html#request-level
+.. _Query-level errors: https://doc.scrapinghub.com/autoextract.html#query-level
+
+Retries
+~~~~~~~
+
+By default, we will automatically retry Network and Request-level errors.
+You could also enable Query-level errors retries
+by specifying the ``--max-query-error-retries`` argument.
```

Comment:
Maybe ``--max-query-error-retries=<N RETRIES>``

Reply:
This is how arguments are referenced by other parts of the readme file. 
I also think that these arguments are well covered by the command-line utility's help texts.